connector/server/survivalanalysis/survival_query.go

```diff
@@ -156,49 +156,58 @@ func (q *Query) Execute() error {
 			logrus.Infof("successful I2B2 explore query %d", i)
```

Comment:
The log messages here need to have more context. Remember that when shown in the console everything is mixed up. E.g. adding a prefix "Survival analysis <query ID>: ..."